### DISCUSSION TITLE
.Net core 3.1 Intermittently hangs

### DISCUSSION THREAD

**Discussion by saurabhvats007** [dotnet/runtime#49311] [Q&A] [answered]
we recently migrated our api from .net framework 4.7 to .net core 3.1. when we moved this to production we noticed that application simply hangs after running for few minutes/hours under high load. when I checked  hang dump I noticed the threads are all stuck in connecthelper connectAsync method of system.net.http . noticed that issue looks to be same as mentioned here

https://github.com/dotnet/runtime/issues/42198

Fix is marked for release with v 6.0.0.0 , is there a plan to merge this fix with dotnet runtime 3.1 for people who are using earlier version.

**Comment by huoyaoyuan** [comment, marked as answer]
You can ask for a backport request, and the servicing team will examine the quality and risk. @geoffkizer ?
